pkg/platform/runtime/artifactcache/artifactcache_test.go

```diff
@@ -0,0 +1,114 @@
+package artifactcache
```

Comment:
Since this is reaching out to the filesystem we should mark this as an integration test (ie. rename it to *_int_test.go).

Alternatively we could segment out the parts where we do IO such that the unit tests can run without needing IO.